docs(configuration): warn users about the usage of asynchronous global scripts (#1350)
ionic-team/stencil-site: docs/config/01-overview.md

```

:::note

> The exported function can also be `async` but be aware that this can have implications on the performance of your application as all rendering operations are being executed after the global script finishes.

Reviewer: ```suggestion
The exported function can also be `async` but be aware that this can have implications on the performance of your application as all rendering operations will be deferred until after the global script finishes.
```

just one possible wording suggestion here
